src/crimson/osd/objclass.cc

```diff
@@ -293,6 +293,11 @@ int cls_cxx_map_get_vals(cls_method_context_t hctx,
 
 int cls_cxx_map_read_header(cls_method_context_t hctx, bufferlist *outbl)
 {
+  OSDOp op{ CEPH_OSD_OP_OMAPGETHEADER };
```

Comment:
```suggestion
  OSDOp op{CEPH_OSD_OP_OMAPGETHEADER};
```

Reply:
@AmnonHanuhov i think you addressed this comment in another commit, see eba2c2a0cf95370bc4fad3d9c49c0ff55943480a  .

could you squash the related changes into a single commit, so the commits are atomic and self-contained?

the same applies to some other changes in https://github.com/ceph/ceph/pull/35275/commits/eba2c2a0cf95370bc4fad3d9c49c0ff55943480a